nvdaHelper/common/ia2utils.cpp

```diff
@@ -74,6 +74,28 @@ void IA2AttribsToMap(const wstring &attribsString, map<wstring, wstring> &attrib
 	}
 }
 
+std::pair<std::vector<CComVariant>, HRESULT>
+getAccessibleChildren(IAccessible* pacc, long indexOfFirstChild, long maxChildCount) {
+	std::vector<CComVariant> varChildren(maxChildCount);
+	const auto res = AccessibleChildren(
+		pacc,
+		indexOfFirstChild,
+		maxChildCount,
+		varChildren.data(),
+		&maxChildCount
+	);
+	if (res != S_OK) {
+		return std::make_pair(
+			std::vector<CComVariant>(0),
+			res
+		);
+	}
+	// shrink the vector in case less children were returned.
+	varChildren.resize(maxChildCount); // so that varChildren.size() will equal actual filled size
+	varChildren.shrink_to_fit(); // so that excess capacity isn't kept

```

Comment:
I don't think this is necessary. I'll double check that that the vector capacity is addressed by copying via `make_pair`.